india_compliance/gst_india/utils/__init__.py

```diff
@@ -121,6 +121,27 @@ def get_party_for_gstin(gstin, party_type="Supplier"):
         return party[0][0]
 
 
+@frappe.whitelist()
+def get_party_contact_details(party, party_type="Supplier"):
+    if not party:
+        return
+
+    filters = [
+        ["Dynamic Link", "link_doctype", "=", party_type],
+        ["Dynamic Link", "link_name", "=", party],
+    ]
+
+    contact = frappe.db.get_value("Contact", filters=filters)
```

Comment:
What if there are more than 1 address?
--> You should ideally return the primary contact / billing contact / default contact...